extensions-contrib/moving-average-query/src/test/resources/runtime.properties

```diff
@@ -0,0 +1,58 @@
+druid.broker.cache.unCacheable=["groupBy","segmentMetadata"]
```

Comment:
Probably this file is not necessary?

Reply:
I do need this file for tests to succeed, I removed all unrequired properties.